src/components/MainView/MainView.jsx

```diff
@@ -26,50 +30,149 @@ export const MainView = () => {
             Genre: doc.Genre.Name,
             Director: doc.Director.Name,
             Image: doc.ImagePath,
-            id: doc._id
+            id: doc._id,
           };
         });
         setMovies(MoviesFromApi);
       });
   }, [token]);
 
-  if (!user) {
-    return (
-      <>
-        <LoginView onLoggedIn={(user, token) => {
-          setUser(user);
-          setToken(token);
-        }} />
-        or
-        <SignupView />
-      </>
-    );
-  }
-
-  if (selectedMovie) {
-    return (
-      <><button onClick={() => { setUser(null); setToken(null); localStorage.clear(); }}>Logout</button>
```

Comment:
Better to have a separate function for this

Reply:
Red means erased.